README.rst

```diff
@@ -94,12 +94,34 @@ To check that a file is correctly sorted use the `--check` flag.
 To allow ``ssort`` to rearrange your file, simply invoke with no extra flags.
 If ``ssort`` needs to make changes to a `black <https://black.readthedocs.io/en/stable/>`_ conformant file, the result will not necessarily be `black <https://black.readthedocs.io/en/stable/>`_ conformant.
 The result of running `black <https://black.readthedocs.io/en/stable/>`_ on an ``ssort`` conformant file will always be ``ssort`` conformant.
-We recommend that you reformat using `isort <https://pycqa.github.io/isort/>`_ and `black <https://black.readthedocs.io/en/stable/>`_ immediately _after_ running ``ssort``.
+We recommend that you reformat using `isort <https://pycqa.github.io/isort/>`_ and `black <https://black.readthedocs.io/en/stable/>`_ immediately after running ``ssort``.
 
 .. code:: bash
 
     $ ssort src/ tests/ setup.py; isort src/ tests/ setup.py; black src/ tests/ setup.py
 
+Pre-commit config example:
```

Comment:
I think this needs a bit more detail.  At the very least, a link to the pre-commit hooks manager package documentation, and some indication that this is optional.

Reply:
like this?

```rst
or you can setup `pre-commit <https://pre-commit.com/index.html>` hook, for example:```